docs/sftt/sftt-api-spec.md

```diff
@@ -232,15 +234,15 @@ Create a team. The team will only contain the member that created it who is now
 
 ##### `200 OK`
 
-!!! missing "Should return the same as the get team route"
+Returns the same response as the get teams route for the newly created team.
```

Comment:
I know this is very small but it should probably be the same response as the "Get a Team" route

Reply:
Fair, considered it already